docs/product/alerts/uptime-monitoring/index.mdx

```diff
@@ -28,7 +26,7 @@ allowing you to address the underlying connectivity problems.
 ## Notifications
 
 An uptime alert continuously monitors the configured URL with the criteria defined above. If a failure is detected,
-a new [uptime issue](/product/issues/issue-details/uptime-issues/) with failed check and related errors details will be created.
+a new [uptime issue](/product/issues/issue-details/uptime-issues/) with failed check and related errors details will be created. To avoid triggering false alerts due to transient issues like network glitches, a new issue is created only after detecting a minimum of three consecutive failures following the initial downtime detection.
```

Comment:
```suggestion
a new [uptime issue](/product/issues/issue-details/uptime-issues/) with the failed check and related error details will be created. To avoid triggering false alerts due to transient issues like network glitches, a new issue is created only after detecting a minimum of three consecutive failures following the initial downtime detection.
```